src/main/java/com/dataguise/saas/dto/SchedulerDTO.java

```diff
@@ -0,0 +1,170 @@
+
+    public void setSourceModule(String sourceModule) {
+        this.sourceModule = sourceModule;
+    }
+
+    private String scheduleType = null;
+
+    private List<String> scheduleDay = null;
+
+    //ISO DATE FORMAT
+    private String startDateTime = null;
+
+    //Weekly/Daily using hourly interval 24:00
+    private String scanType = null;
+
+    private String scheduleInterval;
+
+    //For now we fix this. No need this from UI.
+    private String scheduleScanEndDate = null;
+
+    private Boolean scheduledEventEnabled = null;
+
+    private Integer id = null;
+
+    private String moduleType = null;
+
+    @JsonIgnore
+    public List<TaskParamsDTO> getTaskParams() {
+        return taskParams;
+    }
+
+    @JsonProperty
+    public void setTaskParams(List<TaskParamsDTO> taskParams) {
+        this.taskParams = taskParams;
+    }
+
+    @JsonIgnore
+    public String getScheduleName() {
+        return scheduleName;
+    }
+
+    @JsonProperty
+    public void setScheduleName(String scheduleName) {
+        this.scheduleName = scheduleName;
+    }
+
+    @JsonIgnore
+    public String getScheduleType() {
+        return scheduleType;
+    }
+
+    @JsonProperty
+    public void setScheduleType(String scheduleType) {
+        this.scheduleType = scheduleType;
+    }
+
+    @JsonIgnore
+    public String getScheduleInterval() {
+        return scheduleInterval;
+    }
+
+    @JsonProperty
+    public void setScheduleInterval(String scheduleInterval) {
+        this.scheduleInterval = scheduleInterval;
+    }
+
+    @JsonIgnore
+    public Integer getId() {
+        return id;
+    }
+
+    @JsonProperty
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    @JsonIgnore
+    public String getModuleType() {
+        return moduleType;
+    }
+
+    @JsonProperty
+    public void setModuleType(String moduleType) {
+        this.moduleType = moduleType;
+    }
+
+    @JsonIgnore
+    public List<String> getScheduleDay() {
+        return scheduleDay;
+    }
+
+    @JsonProperty
+    public void setScheduleDay(List<String> scheduleDay) {
+        this.scheduleDay = scheduleDay;
+    }
+
+    @JsonIgnore
+    public String getScheduleScanEndDate() {
+        return scheduleScanEndDate;
+    }
+
+    @JsonProperty
+    public void setScheduleScanEndDate(String scheduleScanEndDate) {
+        this.scheduleScanEndDate = scheduleScanEndDate;
+    }
+
+    @JsonIgnore
+    public String getStartDateTime() {
+        return startDateTime;
+    }
+
+    @JsonProperty
+    public void setStartDateTime(String startDateTime) {
+        this.startDateTime = startDateTime;
+    }
+
+    @JsonIgnore
+    public String getScanType() {
+        return scanType;
+    }
+
+    @JsonProperty
+    public void setScanType(String scanType) {
+        this.scanType = scanType;
+    }
+
+    public DgTaskScheduleInfoBean getSchedule() {
+        return schedule;
+    }
+
+    public void setSchedule(DgTaskScheduleInfoBean schedule) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SchedulerDTO.java L152
    
    &#39;schedule&#39; hides a field.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-368120497>*